src/redux/actions/componentConfig/savePlotConfig.js

```diff
@@ -3,8 +3,11 @@ import getApiEndpoint from '../../../utils/apiEndpoint';
 
 const savePlotConfig = (experimentId, plotUuid) => async (dispatch, getState) => {
   // Do not save the 'outstandingChanges' state to the database.
-  const { outstandingChanges, ...content } = getState().plots[plotUuid];
-
+  console.log('SAVING!');
+  const { outstandingChanges, ...content } = getState().componentConfig[plotUuid];
+  // api expects the 'type' parameter, so it has to be set
+  // property type is required in request.body
+  content.type = content.plotType;
```

Comment:
With this we are sending both content.type and .plotType, right? 
Is there any chance sending extra properties might cause problems (for example with the hash we use to cache requests)?